e2e/src/pages/ArticlePage.ts

```diff
@@ -0,0 +1,35 @@
+import { Page } from '@playwright/test';
+import BasePage from '../base/BasePage';
+
+export default class ArticlePage extends BasePage {
+    private selectors = {
+        title: '#page-title',
+        date: '//*[contains(text(), "February 3, 2025")]',
+        readTime: '//*[contains(text(), "5 min read")]',
+    };
+
+    constructor(page: Page) {
+        super(
+            page,
+            'Article Page',
+            'javascript-ui-test-automation-frameworks-latest-updates',
```

Comment:
It's the same here. If you need to open another post and want to use the ArticlePage class, will your code work?